docs/proposals/dramatically-simplify-cluster-creation.md

```diff
@@ -0,0 +1,258 @@
+
+    * @smarterclayton
+
+## Work streams – can land after 1.4 feature freeze
+
+1. [Debs](https://github.com/kubernetes/release/pull/35) and [RPMs](https://github.com/kubernetes/release/pull/50) (and binaries?) - so that kubernetes can be installed in the first place
+
+    * @mikedanese & @dgoodwin
+
+1. [kubeadm implementation](https://github.com/lukemarsden/kubernetes/tree/kubeadm-scaffolding) - the kubeadm CLI itself, will get bundled into "alpha" kubeadm packages
+
+    * @lukemarsden & @errordeveloper
+
+1. [Implementation of JWS server](https://github.com/jbeda/kubernetes/blob/discovery-api/docs/proposals/super-simple-discovery-api.md#method-jws-token) from [#30707](https://github.com/kubernetes/kubernetes/pull/30707) - so that we can implement the simple UX with no dependencies
+
+    * @jbeda & @philips?
+
+1. Documentation - so that new users can see this in 1.4 (even if it’s caveated with alpha/experimental labels and flags all over it)
+
+    * @lukemarsden
+
+1. `kubeadm` alpha packages
+
+    * @lukemarsden, @mikedanese, @dgoodwin
+
+### Nice to have
+
+1. [Kubectl apply --purge](https://github.com/kubernetes/kubernetes/pull/29551) - so that addons can be maintained using k8s infrastructure
+
+    * @lukemarsden & @errordeveloper
+
+## kubeadm implementation plan
+
+Based on [@philips' comment here](https://github.com/kubernetes/kubernetes/pull/30361#issuecomment-239588596).
+The key point with this implementation plan is that it requires basically no changes to kubelet except [#30515](https://github.com/kubernetes/kubernetes/issues/30515).
+It also doesn't require kubelet to do TLS bootstrapping - kubeadm handles that.
+
+### kubeadm init master
+
+1. User installs and configures kubelet to look for manifests in `/etc/kubernetes/manifests`
+1. API server CA certs are generated by kubeadm
+1. kubeadm generates pod manifests to launch API server and etcd
+1. kubeadm pushes replica set for prototype jsw-server and the JWS into API server with host-networking so it is listening on the master node IP
+1. kubeadm prints out the IP of JWS server and JWS token
+
+### kubeadm join node --token IP
+
+1. User installs and configures kubelet to have a kubeconfig at `/var/lib/kubelet/kubeconfig` but the kubelet is in a crash loop and is restarted by host init system
+1. kubeadm talks to jws-server on IP with token and gets the cacert, then talks to the apiserver TLS bootstrap API to get client cert, etc and generates a kubelet kubeconfig
+1. kubeadm places kubeconfig into `/var/lib/kubelet/kubeconfig` and waits for kubelet to restart
+1. Mission accomplished, we think.
+
+## See also
+
+* [Joe Beda's "K8s the hard way easier"](https://docs.google.com/document/d/1lJ26LmCP-I_zMuqs6uloTgAnHPcuT7kOYtQ7XSgYLMA/edit#heading=h.ilgrv18sg5t) which combines Kelsey's "Kubernetes the hard way" with history of proposed UX at the end (scroll all the way down to the bottom).
```

Comment:
Add a link for Kelsey's doc? Also, it may be better to use github handles rather than full names.